packages/fullscreenView/components/FullscreenView.tsx

```diff
@@ -19,29 +17,30 @@ export interface FullscreenViewProps {
   /** Whether we automatically add padding to the body of the modal. */
   isContentFlush?: boolean;
   /** Custom header content component. ⚠️Use rarely and with caution⚠️ */
-  headerComponent?: React.ReactNode;
+  headerComponent?: React.JSXElementConstructor<any>;
   /** Custom banner component that will appear above the header. */
   bannerComponent?: React.ReactNode;
   /** Function that gets called when the modal is closed */
   onClose: (event?: React.SyntheticEvent<HTMLElement>) => void;
   /** The base of the `data-cy` value. This is used for writing selectors in Cypress. */
   cypressSelectorBase?: string;
+  children?: React.ReactNode;
 }
 
-class FullscreenView extends React.PureComponent<FullscreenViewProps, {}> {
-  public render() {
-    const {
-      children,
-      ctaButton,
-      closeText,
-      isContentFlush,
-      onClose,
-      title,
-      subtitle,
-      bannerComponent,
-      headerComponent,
-      cypressSelectorBase = "fullscreenView"
-    } = this.props;
+const FullscreenView = React.memo(
```

Comment:
❓ Why do we `memo` here without an dependency array?

> If no array is provided, a new value will be computed on every render.

https://reactjs.org/docs/hooks-reference.html#usememo

Reply:
React.memo !== React.useMemo 

https://reactjs.org/docs/react-api.html#reactmemo

Reply:
React memo recreates the `PureComponent` behavior

Reply:
👍